src/modules/navigation/components/Navigation/Navigation.consts.tsx

```diff
@@ -236,7 +236,7 @@ const getCpAdminManagementDropdown = (
 		{
 			node: renderLink(
 				i18n.t('modules/navigation/components/navigation/navigation___beheer'),
-				'',
+				`/${ROUTE_PREFIXES.beheer}/${ROUTE_PARTS.visitRequests}`,
 				{
 					className: linkCls(['u-color-black', 'u-color-white:md']),
```

Comment:
```suggestion
					className: linkCls(['u-color-black', 'u-color-white:md', styles['c-navigation__link--dropdown']]),
```

Hover effect in de mobile dropdown